addons/stock/migrations/11.0.1.1/post-migration.py

```diff
@@ -275,6 +275,26 @@ def recompute_stock_move_line_qty_different_uom(env):
         )
 
 
+def fill_stock_move_line_consume_rel(cr):
```

Comment:
You have forgotten to remove this from here.

Reply:
I haven't forgotten. Why should I remove it?

Reply:
Then what you have forgotten is to move this to `mrp` module :wink: This shouldn't be here if there's no data feeding this unless you install `mrp`. Thus, we should move it to that module although fields are defined in this one.

Reply:
Oh, right. Removed.